gdmix-workflow/setup.py

```diff
@@ -24,14 +23,15 @@
                  "Intended Audience :: Developers",
                  "License :: OSI Approved"],
     license='BSD-2-CLAUSE',
-    version=f'{VERSION}',
+    version=VERSION,
     package_dir={'': 'src'},
     packages=find_namespace_packages(where='src'),
     package_data={'': ['*.yaml']},
     include_package_data=True,
     install_requires=[
         "setuptools>=41.0.0",
-        f"gdmix-trainer=={VERSION}",
+        "gdmix-trainer==0.3.0",
```

Comment:
why hard code to 0.3.0 but bump the VERSION?

Reply:
Decoupled the version between the workflow and trainer.
I don't see a need to do that, and this way you don't have to worry about the version mismatch and just let the workflow to pull the correct trainer version for you.

Reply:
I think it's hard to decouple the trainer and workflow. Updating workflow without updating trainer is fine, but not vice versa. Actually updating trainer is expected to be much more frequently than workflow, and a new workflow has to be released in order to pick up the new trainer, we try to match the trainer and workflow versions.

Reply:
Resolved, each gdmix-workflow pairs with a specific gdmix-trainer version.